### PR TITLE
Fix breadcrumbs a11y issues

### PR DESCRIPTION
**Pull request by kum9748ar**
## Description
Improved breadcrumbs

## Related Issue
[Breadcrumbs a11y improvements](https://github.com/ethereum/ethereum-org-website/issues/5791#)

### CODE REVIEW

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks for the PR @kum9748ar 💪

Looks like the styling has changed (see screenshot below). Could you amend this so that the style is the same as we currently have?

**Changes proposed:**
Screenshot 2022-04-04 at 14 05 40

**Current styling:**
Screenshot 2022-04-04 at 14 05 44

File: src/components/Breadcrumbs.js [hunk 1]
```diff
@@ -79,21 +79,25 @@ const Breadcrumbs = ({ slug, startDepth = 0, className }) => {
...
-            </CrumbLink>
-            {idx < crumbs.length - 1 && <Slash>/</Slash>}
-          </Crumb>
-        </ListItem>
-      ))}
-    </List>
+    <nav aria-label="Breadcrumb">
+      <List className={className} dir={"auto"}>
+        {crumbs.map((crumb, idx) => (
+          <ol>
```
Comment (pettinarip) [inline src/components/Breadcrumbs.js:85]: `ol` should not be here. It should be wrapping all the crumbs.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Hi @kum9748ar, left a couple of things there to fix. Check out the screenshot shared by @minimalsm and make sure to keep the current styling.

**Comment by kum9748ar** [comment]
@pettinarip @minimalsm thanks for your feedback I will fix them

**Comment by kum9748ar** [comment]
Should I close this pull request ?

**Review by pettinarip — APPROVED** [review]
Looks good! 💪🏼 ty @kum9748ar

**Comment by minimalsm** [comment]
Oops. Accidently didn't add the styling to the slash, which is giving us a different line height than production.

**Comment by kum9748ar** [comment]
@minimalsm thanks for fixing and helping me out

**Review by minimalsm — APPROVED** [review]
All fixed!

[Image from Gyazo](https://gyazo.com/f2966aadc9e382ff0fa5288f6686de55)

Merging, thanks @kum9748ar ❤️

**Pull request merged**